cypress/e2e/14-data-transformation-expressions.cy.ts

```diff
@@ -16,7 +16,7 @@ describe('Data transformation expressions', () => {
 		cy.window()
 			// @ts-ignore
 			.then(
-				(win) => win.onBeforeUnload && win.removeEventListener('beforeunload', win.onBeforeUnload),
+				(win) => win.onbeforeunload && win.removeEventListener('beforeunload', win.onbeforeunload),
```

Comment:
this was intentional.. maybe the name needs to be more unique here and a comment https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/views/NodeView.vue#L2597
goal was to block the browser redirect prevention which breaks tests running in headed mode

Reply:
Yy, I didn't remove it, only fixed the capitalization typo. The event should be `onbeforeunload` not `onBeforeUnload`

Reply:
Oh, I didn't realize it was our method. Reverted.